inst/teal_slices.yml

```diff
@@ -18,13 +18,23 @@ schemas:
             varname:
               type: string
             choices:
-              type: array
-              items:
-                type: ['null', string, boolean, number]
+              anyOf:
+                - 'null'
+                - string
+                - boolean
+                - number
+                - array
+                  items:
+                    type: ['null', string, boolean, number]
```

Comment:
😎

Reply:
array could also be of type array, so we are listing all besides the array : P